### DISCUSSION TITLE
1 Time 100%

### DISCUSSION THREAD

**Discussion by jonesjsc** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#99] [Ideas]
Great tool
It would be neat if I could press a button for a 1 time top off.
Use case is that i run aldente at 805 - but i know i want a cigar later today, so I

1. Have to set battery level to 100%
2. use my laptop on battery outside to smoke my stogie
3. Run my tired 400 cycle battery pack to the end where it often dies
4. when i plug it back in I'm done - but - the charge starts back up and 
5. i have top remember to open up / power on and change the battery level back to 80%

With the 1 time 100% option i could just plug the dead laptop back in and make sure i was on and aldente running and i wouldn't have to fiddle with the tool to remember to reset to 80%

Just a suggestion!

It would be nice to be able to press a button to have the battery charge to 100%, and then maybe take some action (notification) - and then stay at 100% until i disconnect.  Then when I disconnect, it will revert

**Comment by MatthiasKerbl** [comment]
Hi jonesjsc,

sounds like an interesting feature and I will discuss it with David.
Maybe we should call it something like "Stogie Mode".
